### ISSUE TITLE
Mistake in Loss function! ("rpn_loss_cls" function in "losses.py")

### ISSUE THREAD

**Issue by RohanUBagewadi** [kentaroy47/frcnn-from-scratch-with-keras#87]
I think the parameters in for calculating binary_crossentropy should be interchanged i.e
K.binary_crossentropy(y_pred[:, :, :, :], y_true[:, :, :, num_anchors:]) instead should be
K.binary_crossentropy(y_true[:, :, :, num_anchors:], y_pred[:, :, :, :]).

According to [Keras ](https://keras.io/api/losses/probabilistic_losses/#binarycrossentropy-class) documentation the first parameter should be target i.e Y_true and second should be the predicted value Y_pred.

Please check is this issue is valid or not.

**Comment by TiasM** [comment]
I also need to confirm about the RPN Architecture. 

x_class = Convolution2D(num_anchors, (1, 1), activation='sigmoid', kernel_initializer='uniform', name='rpn_out_class')(x)

shouldn't the number of kernel/fitler equals to num_anchors*2 according to the original paper ? Can someone please confirm this ?
